packages/astro-font/utils.ts

```diff
@@ -11,25 +13,29 @@ interface Record {
 
 interface Source {
   path: string
-  css?: Record
-  style: string
+  css?: astroHTML.JSX.CSSProperties
+  // https://developer.mozilla.org/en-US/docs/Web/CSS/font-style
+  style: "normal" | "italic" | "oblique" | `oblique ${number}deg`
   preload?: boolean
-  weight?: string | number
+  // https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight
+  weight?: "normal" | "bold" | "lighter" | "bolder" | "inherit" | "initial" | "revert" | "revert-layer" | "unset" | 100 | 200 | 300 | 400 | 500 | 600 | 700 | 800 | 900 | number
 }
 
 interface Config {
   name: string
   src: Source[]
   fetch?: boolean
-  display: string
+  // https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display
+  display: "auto" | "block" | "swap" | "fallback" | "optional"
   selector?: string
   preload?: boolean
   cacheDir?: string
   basePath?: string
   fallbackName?: string
   googleFontsURL?: string
   cssVariable?: string | boolean
-  fallback: 'serif' | 'sans-serif'
+  // https://developer.mozilla.org/fr/docs/Web/CSS/font-family
+  fallback: 'serif' | 'sans-serif' | 'monospace' | 'cursive' | 'fantasy' | 'system-ui' | 'emoji' | 'math' | 'fangsong' | 'inherit' | 'initial' | 'unset'
```

Comment:
all of these are great, but we'd need to have the support for these.

Right now, https://github.com/rishi-raj-jain/astro-font/blob/master/packages/astro-font/font.ts#L8-L18 we only support sans-serif and serif.